pom.xml

```diff
@@ -196,6 +196,16 @@
                 <version>2.3.7</version>
                 <extensions>true</extensions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.0</version>
+                <configuration>
+                    <optimize>true</optimize>
```

Comment:
Why <optimize>true</optimize> ? Because it was on James project. I copied the inherited plugin here.